src/main/java/com/googleinterns/zoomtube/servlets/TranscriptServlet.java

```diff
@@ -25,95 +25,29 @@
 import com.google.appengine.api.datastore.Query.FilterOperator;
 import com.google.appengine.api.datastore.Query.FilterPredicate;
 import com.google.appengine.api.datastore.Query.SortDirection;
-import com.google.common.annotations.VisibleForTesting;
 import com.google.common.collect.ImmutableList;
 import com.google.gson.Gson;
 import com.googleinterns.zoomtube.data.TranscriptLine;
 import com.googleinterns.zoomtube.utils.LectureUtil;
 import com.googleinterns.zoomtube.utils.TranscriptLineUtil;
 import java.io.IOException;
-import java.net.URL;
-import java.util.Optional;
+
 import javax.servlet.ServletException;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
-import javax.xml.parsers.DocumentBuilder;
-import javax.xml.parsers.DocumentBuilderFactory;
-import javax.xml.parsers.ParserConfigurationException;
-import org.apache.commons.text.StringEscapeUtils;
-import org.w3c.dom.Document;
-import org.w3c.dom.Element;
-import org.w3c.dom.Node;
-import org.w3c.dom.NodeList;
-import org.xml.sax.SAXException;
 
 /**
  * Provides the transcript for a given lecture.
  */
 public class TranscriptServlet extends HttpServlet {
-  private static final String XML_URL_TEMPLATE = "http://video.google.com/timedtext?lang=en&v=";
-  public static final String ATTR_START = "start";
-  public static final String ATTR_DURATION = "dur";
-  public static final String TAG_TEXT = "text";
-
   private DatastoreService datastore;
 
   @Override
   public void init() throws ServletException {
     datastore = DatastoreServiceFactory.getDatastoreService();
   }
 
-  @Override
-  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    String videoId = request.getParameter(LectureUtil.VIDEO_ID);
-    Document document = getTranscriptXmlAsDocument(videoId).get();
-    long lectureId = Long.parseLong(request.getParameter(LectureUtil.ID));
-    Key lectureKey = KeyFactory.createKey(LectureUtil.KIND, lectureId);
-    putTranscriptLinesInDatastore(lectureKey, document);
```

Comment:
Why was all this code removed? Can it be done in a later PR?

Reply:
Was done in a previous PR